instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/patches/opentelemetry_graphql_tracing.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module GraphQL
+      module Patches
+        # OpenTelemery Graphql platform_tracing implementation for instrumentation
+        class OpenTelemetryGraphQLTracing < ::GraphQL::Tracing::PlatformTracing
+          self.platform_keys = {
+            'lex' => 'lex.graphql',
+            'parse' => 'parse.graphql',
+            'validate' => 'validate.graphql',
+            'analyze_query' => 'analyze.graphql',
+            'analyze_multiplex' => 'analyze.graphql',
+            'execute_multiplex' => 'execute.graphql',
+            'execute_query' => 'execute.graphql',
+            'execute_query_lazy' => 'execute.graphql'
+          }
+
+          def platform_trace(platform_key, key, data)
+            tracer.in_span(
+              platform_key,
+              attributes: {},
+              kind: :client
+            ) do |span|
+              if key == 'execute_multiplex'
+                operations = data[:multiplex].queries.map(&:selected_operation_name).join(', ')
+                span.set_attribute('operation', operations)
+              end
+
+              if key == 'execute_query'
+                span.set_attribute(:selected_operation_name, data[:query].selected_operation_name)
+                span.set_attribute(:selected_operation_type, data[:query].selected_operation.operation_type)
```

Comment:
```suggestion
                span.set_attribute(:selected_operation_type, data[:query].selected_operation&.operation_type)
```
From our code, it looks like there's a chance that `selected_operation` is `nil`.